#! /bin/sh -e
## 30_library_map_freeproc.dpatch by Craig Small <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Added freeproc to library.map

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argum
ent"
                exit 1;;
esac

exit 0
@DPATCH@
--- procps-3.2.0.orig/proc/library.map
+++ procps-3.2.0/proc/library.map
@@ -4,7 +4,7 @@
 
   readproc; readtask; readproctab; readproctab2; look_up_our_self; escape_command;
   escape_str; escape_strlist;
-  openproc; closeproc;
+  openproc; closeproc; freeproc;
   tty_to_dev; dev_to_tty; open_psdb_message; open_psdb; wchan;
   display_version; procps_version; linux_version_code;
   Hertz; smp_num_cpus;
